### PR TITLE
Fixes example code in docstring of pandas.DataFrame.query

### LINKED ISSUE #21733: Error in docstring for pandas/core/frame.py
There is an error in the docstring for the query function in

`pandas/core/frame.py`.

The PR here fixes this: https://github.com/pandas-dev/pandas/pull/21731

#### Code Sample, a copy-pastable example if possible

```python
# Your code here

```
#### Problem description

[this should explain **why** the current behaviour is a problem and why the expected output is a better solution.]

**Note**: We receive a lot of issues on our GitHub tracker, so it is very possible that your issue has been posted before. Please check first before submitting so that we do not have to handle and close duplicates!

**Note**: Many problems can be resolved by simply upgrading `pandas` to the latest version. Before submitting, please check if that solution works for you. If possible, you may want to check if `master` addresses this issue, but that is not necessary.

For documentation-related issues, you can check the latest versions of the docs on `master` here:

https://pandas-docs.github.io/pandas-docs-travis/

If the issue has not been resolved there, go ahead and file it in the issue tracker.

#### Output of ``pd.show_versions()``

<details>

[paste the output of ``pd.show_versions()`` here below this line]

</details>

### PR DESCRIPTION
**Pull request by alphaCTzo7G**
- [ ] closes #21733
- [ ] tests added / passed
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/core/frame.py [hunk 1]
```diff
@@ -2841,7 +2841,7 @@ def query(self, expr, inplace=False, **kwargs):
         --------
         >>> from numpy.random import randn
```
Comment (jreback) [inline pandas/core/frame.py]: ahh these are the old forms, can you rather amend to
```
import numpy as np
import pandas as pd
```
then change the randn call

Comment (alphaCTzo7G) [reply pandas/core/frame.py]: @jreback Ok.. I changed it as follows..

        >>> import numpy as np
        >>> import pandas as pd
        >>> df = pd.DataFrame(np.random.randn(10, 2), columns=list('ab'))
        >>> df.query('a > b')
        >>> df[df.a > df.b]  # same result as the previous expression

I force pushed it to the same branch.. 

https://github.com/alphaCTzo7G/pandas/tree/fix_typo_query

I noticed there are some other docstrings with `from pandas` or `from numpy`..

```
./pandas/plotting/_converter.py|134 col 17| ">>> from pandas.plotting import register_matplotlib_converters"
./pandas/plotting/_misc.py|501 col 5| >>> from pandas import read_csv
./pandas/plotting/_misc.py|502 col 5| >>> from pandas.tools.plotting import parallel_coordinates
./pandas/util/testing.py|602 col 5| >>> from pandas.util.testing import capture_stdout
./pandas/util/testing.py|650 col 5| >>> from pandas.util.testing import capture_stderr
./pandas/util/testing.py|2213 col 7| >>> from pandas.util.testing import network
./pandas/util/testing.py|2214 col 7| >>> from pandas.io.common import urlopen
./pandas/tseries/holiday.py|145 col 9| >>> from pandas.tseries.holiday import Holiday, nearest_workday
./pandas/tseries/holiday.py|146 col 9| >>> from pandas import DateOffset
pandas/core/sparse/series.py|739 col 9| >>> from numpy import nan
pandas/core/algorithms.py|1519 col 5| >>> from pandas.api.extensions import take
pandas/core/dtypes/concat.py|285 col 5| >>> from pandas.api.types import union_categoricals
./pandas/io/pytables.py|457 col 5| >>> from pandas import DataFrame
./pandas/io/pytables.py|458 col 5| >>> from numpy.random import randn
./pandas/io/json/normalize.py|138 col 5| >>> from pandas.io.json import json_normalize
./pandas/io/stata.py|2713 col 5| >>> from pandas.io.stata import StataWriter117
```

Do these need to be adjusted as well?

Comment (gfyoung) [reply pandas/core/frame.py]: Probably, but don't fix them here.  Would do in a separate PR.

**Review by gfyoung — APPROVED** [review]
LGTM!

cc @jreback

File: pandas/core/frame.py (hunk 1 above)
Comment (alphaCTzo7G) [reply pandas/core/frame.py]: Ok.. I will create 1 PR to fix the rest of them..

**Pull request merged**